src/doc/src/reference/config.md

```diff
@@ -443,6 +444,22 @@ directory.
 
 This option is deprecated and unused. Cargo always has pipelining enabled.
 
+#### `build.timings`
+* Type: array of strings
+* Default: none
+* Environment: `CARGO_BUILD_TIMINGS`
+
+Output information how long each compilation takes, and track concurrency information over time. Possible values:
+
+* `html`: Write a human-readable file `cargo-timing.html` to the
+  `target/cargo-timings` directory with a report of the compilation. Also write
+  a report to the same directory with a timestamp in the filename if you want
+  to look at older runs. HTML output is suitable for human consumption only,
+  and does not provide machine-readable timing data.
+* `json`: Emit machine-readable JSON information about timing information.
```

Comment:
Can you also mention here that `json` is unstable and requires `-Zunstable-options` on the nightly channel?

Reply:
Added.